src/Worker/Core/Logs.cs

```diff
@@ -9,15 +9,35 @@ namespace Microsoft.DurableTask
     /// <summary>
     /// Log messages.
     /// </summary>
-    /// <remarks>
-    /// NOTE: Trying to make logs consistent with https://github.com/Azure/durabletask/blob/main/src/DurableTask.Core/Logging/LogEvents.cs.
-    /// </remarks>
     static partial class Logs
     {
+        /// <summary>
+        /// The common category name for worker logs.
+        /// </summary>
+        internal const string WorkerCategoryName = "Microsoft.DurableTask.Worker";
```

Comment:
Consider using more granular names under this prefix. Such as `Microsoft.DurableTask.Worker.Activities`, `Microsoft.DurableTask.Worker.Orchestrations`